data/game.json

```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "publish_date": "2022-12-12",
+    "genre": "sports",
+    "label_title": "Fifa",
+    "label_color": "white",
+    "author_first_name": "mad",
+    "author_last_name": "lab",
+    "multiplayer": true,
+    "last_played_at": "2022-12-12"
+  }
+]
```

Comment:
- Great Job for being able to display the JSON data, I think you forget to save what you added, so please save it to a JSON file to be able to get what we added after quitting the app 👍